src/desktop/camera.rs

```diff
@@ -209,13 +212,7 @@ fn pipewire_streams_inner<F: Fn(Stream) + Clone + 'static, G: FnOnce() + Clone +
 /// *Note* The socket referenced by `fd` must not be used while this function is
 /// running.
 #[cfg(feature = "pipewire")]
-pub async fn pipewire_streams(fd: RawFd) -> Result<Vec<Stream>, pw::Error> {
```

Comment:
this means we can drop the dependency on libc right?

Reply:
Yes, dropped.